C/Chemfiles/build_tarballs.jl

```diff
@@ -3,13 +3,13 @@
 using BinaryBuilder, Pkg
 
 name = "Chemfiles"
-version = v"0.9.3"
+version = v"0.10.0-rc1"
```

Comment:
Please don't use prerelease versions here, they create some troubles to the package manager

Reply:
As mentioned in the first comment of the PR, this is only a WIP PR to check that everything builds. I'll change the version to the full release before asking for a merge 😃